src/realm/exceptions.cpp

```diff
@@ -64,9 +64,12 @@ Exception::Exception(Status status)
 {
 }
 
-Status exception_to_status() noexcept
+Status exception_to_status(std::exception_ptr exc_ptr) noexcept
 {
     try {
+        if (exc_ptr) {
```

Comment:
This doesn't appear to be used anywhere.

Reply:
I was previously using this to get the exception thrown in the Async open start callback, but not anymore. Removed this unused code.